.gitignore

```diff
@@ -1 +1,3 @@
 /target
+.*
```

Comment:
I'm not sure of how this interacts with `.github/` / `.gitignore` / `.git`, but LGTM